8/alpine/Dockerfile

```
@@ -23,7 +23,7 @@
 # FIXME(oleg_nenashev): This is not an official AdoptOpenJDK Docker Image.
 # There is no official Alpine images at the moment.
 # Needs upgrade when/if there is an official alpine image.
-FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u322-b06
+FROM eclipse-temurin:8u322-b06-jdk-alpine
```

Intentionally using the preceding JDK 8u322 rather than the most recent JDK 8u332?
```suggestion
FROM eclipse-temurin:8u322-b06-jdk-alpine
```

---

@MarkEWaite absolutely: I don't want to mess with #118 :) 
If #118 is merged before the current, then I'll update there.